package.json

```diff
@@ -46,13 +45,16 @@
     "eslint-plugin-react": "^7.0.0",
     "eslint-plugin-standard": "^3.0.1",
     "md5-file": "^3.2.3",
+    "minimist": "^1.2.5",
     "mocha": "^5.1.1",
-    "nock": "^9.2.5",
+    "nock": "^12.0.3",
     "should": "6.0.3",
+    "tslint": "^5",
     "typescript": "^2.8.3",
     "uuid": "^3.0.1"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
There is a security vulnerability problem detected in the `minimist`, so `npm audit` command drops a warning if it detects wrong `minimist` version. The automated testing pipeline calls it here:
https://github.com/cloudant/nodejs-cloudant/blob/35dbb085857bb423bc86aef10c4e27a302e1f113/package.json#L56

The problem is that `eslint` and `mocha` packages are using this not secure version. This way the testing pipeline stops at the audit check.
With this `preinstall` command I could set to use a preinstalled `minimist` version.
I used this article as a solution base: https://itnext.io/fixing-security-vulnerabilities-in-npm-dependencies-in-less-than-3-mins-a53af735261d
Before merge I thought to check if there are any corrected package available from them.

Reply:
I could only guess that you changed it because of the pipeline. Thanks for the thorough explanation! You have spent way more time on this solution to not spend more, this solution is fine for me.